### PR TITLE
Cunit test case for mqttcm_webcfg.c

### PR DESCRIPTION
**Pull request by guruchandru**

### CODE REVIEW

File: src/mqttcm_webcfg.c [hunk 1]
```diff
@@ -158,6 +165,7 @@ char * createMqttPubHeader(char * payload, ssize_t * payload_len)
 	char *pub_headerlist = NULL;
 
 	pub_headerlist = (char *) malloc(sizeof(char) * 1024);
+	memset(pub_headerlist, 0, sizeof(char) * 1024);
```
Comment (sadhyama) [inline src/mqttcm_webcfg.c:168]: Add Null check for pub_headerlist and then do memset

File: tests/test_mqttcm_webcfg.c [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+	MqttCMInfo("rbus_open for component %s\n", CONSUMER_COMPONENT_NAME);
+	rc = rbus_open(&handle, CONSUMER_COMPONENT_NAME);
+	if(rc != RBUS_ERROR_SUCCESS)
+	{
+		CU_FAIL("rbus_open failed for subscribe_to_event");
+	}
+
+	if(strncmp(eventname, WEBCFG_MQTT_SUBSCRIBE_CALLBACK, strlen(WEBCFG_MQTT_SUBSCRIBE_CALLBACK)) == 0)
+	{
+		printf("Inside subscribeEventSuccessCallbackHandler for %s and eventname is %s\n", WEBCFG_MQTT_SUBSCRIBE_CALLBACK, eventname);
```
Comment (sadhyama) [inline tests/test_mqttcm_webcfg.c:124]: Use mqttcmInfo instead of printf

File: src/mqttcm_webcfg.c (hunk 1 above)
Comment (guruchandru) [reply src/mqttcm_webcfg.c]: added in new review

File: tests/test_mqttcm_webcfg.c (hunk 2 above)
Comment (guruchandru) [reply tests/test_mqttcm_webcfg.c]: changes are added

**Pull request merged**